### PR TITLE
Add a test for specified depth rendering

### PR DESCRIPTION
**Pull request by gnarf**

### CODE REVIEW

File: src/deep-render.test.js [hunk 1]
```diff
@@ -49,3 +49,13 @@ it(`renders components with null children`, () => {
...
 });
+
+it('renders to a specified depth', () => {
+  const ErrorIfRendered = () => { throw new Error('this should not render'); };
+  const Second = () => <ErrorIfRendered />;
+  const First = () => <div><Second /><Second /></div>;
+  const context = render(<First />, { depth: 2 });
+  expect(context.find('ErrorIfRendered').length).toBe(2);
+
+  expect( () => render(<First />, { depth: 3 })).toThrow();
```
Comment (mzgoddard) [inline src/deep-render.test.js]: Minor nit: space before the arrow function.

Comment (gnarf) [inline src/deep-render.test.js]: Cheers, can we land it with the nit and find the eslint rule to fix it after we land #17 too?

Comment (gnarf) [inline src/deep-render.test.js]: oh wait, inverted!!!

I'll rebase and find the eslint rule ;)

Comment (mzgoddard) [inline src/deep-render.test.js]: Just merged #17. If we didn't have the rule enforcing it, go ahead and merge and add the rule in another pr.

File: .eslintrc.js [hunk 2]
```diff
@@ -46,43 +46,44 @@ module.exports = {
     "jest/no-disabled-tests": 1,
     "jest/no-focused-tests": 1,
     "jest/no-identical-title": 2,
-    "no-empty": 0,
```
Comment (gnarf) [inline .eslintrc.js]: sorry for the noise, I sorted this list to make sure `space-in-parens` wasn't already listed

**Pull request merged**